tests/test_models.py

```diff
@@ -1613,17 +1613,17 @@ def test_validate_categories(self):
 
     def test_validate_bad_doi(self):
         with assert_raises(ValidationError):
-            Node(preprint_doi='nope').save()
+            Node(preprint_article_doi='nope').save()
         with assert_raises(ValidationError):
-            Node(preprint_doi='https://dx.doi.org/10.123.456').save()  # should save the bare DOI, not a URL
+            Node(preprint_article_doi='https://dx.doi.org/10.123.456').save()  # should save the bare DOI, not a URL
         with assert_raises(ValidationError):
-            Node(preprint_doi='doi:10.10.1038/nwooo1170').save()  # should save without doi: prefix
+            Node(preprint_article_doi='doi:10.10.1038/nwooo1170').save()  # should save without doi: prefix
 
     def test_validate_good_doi(self):
         doi = '10.10.1038/nwooo1170'
-        self.node.preprint_doi = doi
+        self.node.preprint_article_doi = doi
         self.node.save()
-        assert_equal(self.node.preprint_doi, doi)
+        assert_equal(self.node.preprint_article_doi, doi)
 
```

Comment:
Added in https://github.com/CenterForOpenScience/osf.io/pull/6346/commits/7c4c0c7a1027e2064e47d5c4e24c3ac65dbc3b58:
* model tests for permissions
* view tests for `NodePreprintsList`
* validator tests for `.set_subjects`